src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionTanInputViewController.swift

```diff
@@ -18,32 +18,57 @@
 import Foundation
 import UIKit
 
-class ExposureSubmissionTanInputViewController: UIViewController, SpinnerInjectable {
+class ExposureSubmissionTanInputViewController: UIViewController, ENANavigationControllerWithFooterChild, SpinnerInjectable {
 	// MARK: - Attributes.
 
 	@IBOutlet var scrollView: UIScrollView!
 	@IBOutlet var contentView: UIView!
 	@IBOutlet var descriptionLabel: UILabel!
 	@IBOutlet var errorLabel: UILabel!
 	@IBOutlet var errorView: UIView!
-	@IBOutlet var tanInput: ENATanInput!
+	@IBOutlet var tanInput: ENATanInput! { didSet { tanInput.delegate = self } }
 
 	var initialTan: String?
 	var exposureSubmissionService: ExposureSubmissionService?
 	var spinner: UIActivityIndicatorView?
 
 	// MARK: - View lifecycle methods.
 
+	override func viewDidLoad() {
```

Comment:
is the plan to merge the PR with the spinners in the buttons before or after this PR?

Reply:
Spinners will merged with #439 when we apply button accessibility.

Reply:
Alright! Thanks for clarifying.